js/src/vyxal/JSVyxal.scala

```diff
@@ -54,6 +54,45 @@ object JSVyxal:
     catch case ex: VyxalException => errorFunc(ex.getMessage(using ctx))
   end execute
 
+  @JSExport
+  def theCoolerExecute(
+      code: String,
+      inputs: js.Array[String],
+      printFunc: js.Function1[String, Unit],
+      errorFunc: js.Function1[String, Unit],
+      presetStack: Boolean = false,
+      endPrintMode: EndPrintMode = EndPrintMode.Default,
+      defaultValue: String = "0",
+      rangify: Boolean = false,
+      rangeStart: Int = 1,
+      rangeOffset: Int = 0,
+      literate: Boolean = false,
+  ): Unit =
+    val inputList =
+      inputs.map(MiscHelpers.eval(_)(using Context())).toSeq.reverse
+    val settings = Settings(
```

Comment:
Maybe I'm just not understanding, but what's the benefit of this over parsing a list of flags like in the old execute?

ex. `val settings = Settings(online = true).withFlags(flags.toList)`

Reply:
It maps better to the UI design I have in mind; beyond that, it gives the frontend more power to configure Vyxal as it wishes.